src/Microsoft.Health.Api/Features/Audit/AuditException.cs

```diff
@@ -3,14 +3,15 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System.Globalization;
 using Microsoft.Health.Abstractions.Exceptions;
 
 namespace Microsoft.Health.Api.Features.Audit;
 
 public class AuditException : MicrosoftHealthException
 {
     public AuditException(string controllerName, string actionName)
-        : base(string.Format(Resources.MissingAuditInformation, controllerName, actionName))
+        : base(string.Format(CultureInfo.CurrentCulture, Resources.MissingAuditInformation, controllerName, actionName))
```

Comment:
Generally any client-facing messages, like those localized via the resource files, should be formatted via `CultureInfo.CurrentCulture`. For example, you can see this pattern in the BCLs via the `SR` class which encapsulates "System Resources." See [Format](https://github.com/dotnet/runtime/blob/4ec7a81108d944f292c7886653fc8f42494adc9d/src/libraries/Common/src/System/SR.cs#L55)

It is called without an `IFormatProvider`, which is the same as "current culture" according to the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.string.format?view=net-6.0#format-and-culture)